PureBasicIDE/AutoComplete.pb

```diff
@@ -1204,7 +1204,7 @@ EndProcedure
 Procedure AutoCompleteWindowEvents(EventID)
   
   If AutoCompleteWindowOpen
-      
+  
```

Comment:
Purebasic IDE fill bank lines with the indentation character. Removing it only add some useless diffs in the commit.